packages/entity-server/src/routes/hierarchy/relationships/ResponseBuilder.ts

```diff
@@ -137,23 +146,12 @@ export class ResponseBuilder {
     return formattedEntitiesByCode;
   }
 
-  isDefaultFilter = (filter: EntityFilter) => {
```

Comment:
Cosmetic but in this case I felt the function made a bit more sense outside the class (as it didn't rely on class members and is kind of generic)